selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/CheckConvertingToMavenProjectTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.miscellaneous;
+
+import static java.lang.String.*;
```

Comment:
As far as I know, the common import is a bad practice. Please import relations directly.